src/main/java/com/loadero/model/File.java

```diff
@@ -0,0 +1,101 @@
+package com.loadero.model;
+
+import com.google.gson.annotations.SerializedName;
+import com.loadero.Loadero;
+import com.loadero.http.ApiResource;
+import com.loadero.http.RequestMethod;
+import java.io.IOException;
+
+/**
+ * Used to read information about a project file.
+ */
+public final class File {
+    private final int id;
+    private final String created;
+    private final String updated;
+    @SerializedName("project_id")
+    private final int projectId;
+    @SerializedName("file_type")
+    private final String fileType;
+    private final String content;
+    private final String name;
+    private final String password;
+
+    File(
+        int id, String created, String updated,
+        int projectId, String fileType, String content, 
+        String name, String password
+    ) {
+        this.id = id;
+        this.created = created;
+        this.updated = updated;
+        this.projectId = projectId;
+        this.fileType = fileType;
+        this.content = content;
+        this.name = name;
+        this.password = password;
+    }
+
+    /**
+     * Retrieves file from Loadero.
+     *
+     * @param fileId ID of the file.
+     * @return {@link File} object.
+     * @throws IOException if request failed.
+     */
+    public static File read(int fileId) throws IOException {
```

Comment:
shouldn't we also add a create method?

Reply:
I added it, but it fails to get the id. 
`An exception occurred while executing the Java class. Can not get final int field com.loadero.model.File.id on com.loadero.model.FileParams` which for some reason I fail to debug..